app/models/import.rb

```diff
@@ -21,11 +21,11 @@ class Import < ApplicationRecord
     :needs_backend_support,
     :unfurled
   ]
-  enum courtesy_notification: [
+  enum :courtesy_notification, [
     :not_required,
     :pending,
     :completed
-  ], _prefix: true
+  ], :prefix => true
```

Comment:
I think you can write

```suggestion
  ], prefix: true
```
instead or is there a reason why we use the old syntax?

Reply:
You're right, that was my old Ruby brain 😅 Thanks for catching that.